python/example/berry/mcmc.py

```diff
@@ -0,0 +1,98 @@
+# flake8: noqa E402
+import os
+
+os.environ["XLA_FLAGS"] = "--xla_force_host_platform_device_count=6"
+import jax
+import jax.numpy as jnp
+import numpy as np
+import numpyro
+import numpyro.distributions as dist
+import numpyro.infer
+
+# This line is critical for enabling 64-bit floats.
+from jax.config import config
+
+config.update("jax_enable_x64", True)
+
```

Comment:
A bunch of changes through here in the MCMC code. They aren't super substantive, but between moving the file and making a bunch of changes, it seems git lost the plot and is treating this as a new file. So, the diff is useless...